### DISCUSSION TITLE
helm chart in argocd template

### DISCUSSION THREAD

**Discussion by ekhaydarov** [renovatebot/renovate#20053] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

34.113.2

### If you're self-hosting Renovate, select which platform you are using.

None

### If you're self-hosting Renovate, tell us what version of the platform you run.

argo workflows

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

parse an argocd file by the name of `appset-config.yaml` which contains
```
template:
  repoURL: https://azure.github.io/azure-workload-identity/charts
  targetRevision: 0.15.0
  chart: azure-workload-identity
```

### What you tried so far.

tried argocd manager, which did not pick it up
tried regex manager 
```
{
      "fileMatch": [
        "(^|/)appset-config.yaml$"
      ],
      "matchStringsStrategy": "combination",
      "matchStrings": [
        "\\s{4}repoUrl: (?<registryUrl>\\S+)",
        "\\s{4}chart: (?<depName>\\S+)",
        "\\s{4}targetRevision: (?<currentValue>\\S+)",
      ],
      "datasourceTemplate": "helm"
    }
```
in packagerules matchpaths for exactly where the file is and thats it and it still couldnt find it.

### Relevant debug logs

<details><summary>Logs</summary>
many of these errors but does not tell me which files to look at

```
DEBUG: Failed to parse helm-values YAML (repository=test, baseBranch=renovate-test)
       "err": {
         "name": "YAMLException",
         "reason": "expected a single document in the stream, but found more",
         "message": "expected a single document in the stream, but found more",
         "stack": "YAMLException: expected a single document in the stream, but found more\n    at load
```

```
DEBUG: Matched 203 file(s) for manager regex:
DEBUG: Matched 203 file(s) for manager argocd:
DEBUG: Found 0 package file(s) 
DEBUG: Manager explicitly enabled in "enabledManagers" config, but found no results. Possible config error?
       "manager": "regex"
       "manager": "argocd"
```
</details>

**Comment by secustor** [comment, marked as answer]
The log messages are not relevant to what you are trying to achieve. The `YAMLException` is referring to `values.yaml` files which you seem to have in your repository which not only contain YAML document but multiple. 

Regarding your original inquiry.  Has your example of `appset-config.yaml` the same indentation as your actual file? If yes then there are to few leading spaces for your regex. Should this not be the case, please create a reproduction repo.

**Reply by ekhaydarov** [reply]
indentation issue
